core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationItem.java

```diff
@@ -57,4 +57,5 @@ public String getSystemProperty() {
     public Object getDefaultValue() {
         return defaultValue;
     }
+
```

Comment:
You like white spaces as well :-D https://github.com/arquillian/smart-testing/pull/233#discussion_r144572746